modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -36,6 +36,17 @@ FINISH
 ----
 | New xref:clauses/finish.adoc[FINISH] clause, which can be optionally used to define a query that returns no result.
 
+a|
+label:syntax[]
+label:new[]
+[source, cypher, role=noheader]
+----
+  RETURN 1 AS a
+UNION DISTINCT
+  RETURN 1 AS a
```

Comment:
Is the indenting of the RETURN clauses intentional?

Reply:
I have seen people do this. UNION is one level up in the language structure compared to RETURN, hence, it is not an unreasonable thing to do. However, I should have checked how the documentation formats UNION and follow that.

So here we go:

```suggestion
RETURN 1 AS a
UNION DISTINCT
RETURN 1 AS a
```